bundles/AdminBundle/Resources/public/js/pimcore/object/quantityvalue/unitsettings.js

```diff
@@ -76,6 +76,9 @@ pimcore.object.quantityValue.unitsettings = Class.create({
                 reader: {
                     type: 'json',
                     rootProperty: 'data'
+                },
+                extraParams: {
+                    limit: 0
```

Comment:
![image](https://github.com/pimcore/pimcore/assets/6014195/171b181b-29fa-4e13-9313-a59e6a06e418)
Not sure if `0` is good, probably `999999` is safer 
```suggestion
                    limit: 999999
```

Reply:
limit 0 is no limit so it is fine.